bar_industry/demo/pos_config.xml

```diff
@@ -1,4 +1,8 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version="1.0" encoding="UTF-8"?>
 <odoo noupdate="1">
-    <function model="pos.config" name="load_onboarding_bar_scenario" />
+	<record id="pos_restaurant.pos_config_main_bar" model="pos.config">
+		<field name="minimal_employee_ids" eval="[(6, 0, [ref('hr_employee_2')])]" />
+		<field name="basic_employee_ids" eval="[(6, 0, [ref('hr_employee_5'), ref('hr_employee_6')])]"/>
+		<field name="advanced_employee_ids" eval="[(6, 0, [ref('hr.employee_admin'), ref('hr_employee_4')])]"/>
```

Comment:
Maybe you could already add admin as advanced in data/ (and other ones here in demo)